includes/class-wc-payments-features.php

```diff
@@ -22,6 +22,7 @@ class WC_Payments_Features {
 	const AUTH_AND_CAPTURE_FLAG_NAME        = '_wcpay_feature_auth_and_capture';
 	const PROGRESSIVE_ONBOARDING_FLAG_NAME  = '_wcpay_feature_progressive_onboarding';
 	const DISPUTE_ON_TRANSACTION_PAGE       = '_wcpay_feature_dispute_on_transaction_page';
+	const PAY_FOR_ORDER_FLOW                = '_wcpay_feature_pay_for_order_flow';
```

Comment:
This is unused, perhaps you wanted to use this in line 372

Reply:
Thank you for catching that! Fixed here [4953b3b](https://github.com/Automattic/woocommerce-payments/pull/7296/commits/4953b3b8c8d89d792757d0f69a009d468c9cfb2b)